### ISSUE TITLE
Introduce Peace Time, from Arma2

### ISSUE THREAD

**Issue by SpanishSurfer** [SpanishSurfer/becti_zerty_OFPS.Altis#35]

**Comment by SpanishSurfer** [comment]
Magnet/Live, any chance you can help with this?  

Have a look here to see what i'm talking about....PEACE TIME would be a HUGE benefit for team play.

Run this mission as a local server, play a bit, capture a town near enemy and see what it looks like...Peace time is a great concept from A2.
https://drive.google.com/file/d/0B8g5S2eOid7lWXl0YVlHU3hHWDQ/view?usp=sharing

**Comment by SpanishSurfer** [comment]
I want this update next....it will change game play for the MUCH better...can I have a volunteer?

**Comment by MagnetAss** [comment]
I'll have a look or today, sounds neat!

**Comment by Dain0087** [comment]
This will and can be done but i will need some time to work some things out

**Comment by SpanishSurfer** [comment]
Dain can you do this? If not I'll put time into it.

**Comment by Dain0087** [comment]
Yes, I had to leave for about a month just got my official Discharge from the Army then went on spring break for two weeks, sorry life got crazy but i am back let me look at were i was
